src/lua/fiber.c

```diff
@@ -285,10 +285,6 @@ lbox_fiber_top(struct lua_State *L)
 			      " fiber.top_enable() first");
 	}
 	lua_newtable(L);
-	lua_pushliteral(L, "cpu_misses");
```

Comment:
This counter is [documented](https://www.tarantool.io/en/doc/latest/reference/reference_lua/fiber/#lua-function.fiber.top). You should create a doc bot request to update the documentation.

Reply:
Yes, thats why this pull request is a draft so far. If we agreed on this, I’ll update doc etc.

Reply:
Done.